### PR TITLE
@alloy => Add spacecommander and run it

### PR DESCRIPTION
**Pull request by orta**
Happy to go in and revert a file or two also.

### CODE REVIEW

**Comment by orta** [comment]
Alright, I've reverted the Analytics+DSL file, but for the most of it - I'm pretty happy.

**Comment by orta** [comment]
Alright, this passes locally, and I'm leaving the CocoaPods faffing to @alloy

File: Artsy Tests/ARAppDelegate+Testing.m [hunk 1]
```diff
@@ -27,7 +28,7 @@ + (void)load
     });
 }
 
-+ (void)swapImplementationOf:(SEL)old with:(SEL)new
++ (void)swapImplementationOf:(SEL)old with:(SEL) new
```
Comment (alloy) [inline Artsy Tests/ARAppDelegate+Testing.m]: This is no good.

Comment (orta) [inline Artsy Tests/ARAppDelegate+Testing.m]: agree, I wonder what's up with the one, it's not done it elsewhere.

Comment (alloy) [inline Artsy Tests/ARAppDelegate+Testing.m]: https://github.com/square/spacecommander/issues/14

**Pull request merged**